public/app/features/explore/Explore.tsx

```diff
@@ -229,13 +229,14 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
     );
   }
 
-  renderGraphPanel() {
-    const { graphResult, absoluteRange, timeZone, splitOpen, queryResponse, loading } = this.props;
+  renderGraphPanel(width: number) {
+    const { graphResult, absoluteRange, timeZone, splitOpen, queryResponse, loading, theme } = this.props;
     return (
       <Collapse label="Graph" loading={loading} isOpen>
```

Comment:
I would like to replace this Collapse component with the new PanelChrome https://developers.grafana.com/ui/canary/index.html?path=/story/visualizations-panelchrome--examples   that component aims to be a very generic extensible panel container display only (no logic), with support for title, loading indicators & more, and inner size calculations (no AutoSizer, only based on size passed in).  That way we would get rid of the `{width - theme.panelPadding * 2}` as that is handled internally by PanelChrome. 

But we can't do that right now as PanelChrome does not support collapse/expand states needed for the NodeGraph view